pages/common/ngrep.md

```diff
@@ -0,0 +1,23 @@
+# ngrep
+
+> Grep traffic on a network.
```

Comment:
I use grep because ngrep is like "grep command used in network", but it will confuse for users who have not used grep. Your comment is more meaningful.